pkg/scheduler/api/job_info.go

```diff
@@ -336,6 +336,15 @@ func GetBudget(pg *PodGroup) *DisruptionBudget {
 	return NewDisruptionBudget("", "")
 }
 
+// GetJobMinResources return the min resources of podgroup.
+func (ji *JobInfo) GetJobMinResources(s scheduling.PodGroupSpec) *Resource {
```

Comment:
s/GetJobMinResources/GetMinResources

Let's use the PodGroupSpec of JobInfo, so parameter is not necessary.

Reply:
make sense, also I change the signature `s/getWaitingTime/extractWaitingTime` for they are not getters but extract info from another object